### DISCUSSION TITLE
How to solve wallet duplication？

### DISCUSSION THREAD

**Discussion by 1012492240** [reown-com/appkit#3339] [Q&A]
![image](https://github.com/user-attachments/assets/77761ee6-7a45-4e68-b2c0-3b2d67f677dc)

**Comment by arein** [comment]
Thanks for reporting! We'd love to validate the fix with you. Can you dm me on telegram? "cyberdrk"

**Comment by rtomas** [comment]
Please confirm if you are using the latest version, as it should be fixed.
